api/chunk_compression_stats.md

```diff
@@ -3,6 +3,11 @@
 Get chunk-specific statistics related to hypertable compression.
 All sizes are in bytes.
 
+The data shown by this view is based on cached sizes that are computed
+when `compress_chunk` is executed or when a compression policy touches
+a chunk. For example, an insert into a compressed chunk will not update
+the cached sizes. Refer to `chunks_detailed_size` to compute exact sizes.
```

Comment:
```suggestion
This view shows the cached size of chunks. The cached sizes are computed 
when `compress_chunk` is executed, or when a compression policy touches 
the chunk. An insert into a compressed chunk does not update the cached 
sizes. For more information about how to compute exact sizes, rather than 
cached sizes, see the `chunks_detailed_size` section.
```